lib/features/base/base_mailbox_controller.dart

```diff
@@ -69,6 +69,7 @@ abstract class BaseMailboxController extends BaseController {
   final teamMailboxesTree =  MailboxTree(MailboxNode.root()).obs;
 
   List<PresentationMailbox> allMailboxes = <PresentationMailbox>[];
+  List<PresentationMailbox> listUnsubscribedMailboxes = <PresentationMailbox>[];
```

Comment:
We should not store too many unnecessary variables in memory to avoid wasting memory. Try converting the `allMailboxes` variable to `listUnsubscribedMailboxes` when you need to use it.